examples/example.krun

```diff
@@ -75,5 +75,5 @@ TEMP_READ_PAUSE = 1
 #PRE_EXECUTION_CMDS = ["sudo service cron stop"]
 #POST_EXECUTION_CMDS = ["sudo service cron start"]
 
-# CPU pinning (off by default)
-#ENABLE_PINNING = False
+# CPU pinning (on by default)
+#ENABLE_PINNING = True
```

Comment:
I like manual pages too, at-least for UNIX utilities. The thing is, Krun is not installable per-se, so we won't be placing manual pages in the man path. This would mean to read the manual you would have to do `mandoc /path/to/krun.1 | less` for example.
Kinda awkward, but I guess I could live with it.

Definitely split the docs down into many pages though. One for switches, one for config file, one discussing how to set it up.